components/Navigation/MobileNav/MobileNav.module.scss

```diff
@@ -129,6 +134,27 @@
             left: 14px;
             top: 29px;
         }
+
+        @include hcm() {
+            filter: invert(1);
+        }
+    }
+
+    &:hover {
+        background-color: var(--color--header-icon-color);
+
+        span {
+            background-color: var(--color--white);
+        }
+    }
+
+    @include hcm() {
```

Comment:
@jhancock532 I have just snuck this extra change in as I had missed the underline styles for the links in HCM in the mobile menu